src/globus_sdk/services/transfer/client.py

```diff
@@ -2699,3 +2699,199 @@ def endpoint_manager_delete_pause_rule(
             f"/v0.10/endpoint_manager/pause_rule/{pause_rule_id}",
             query_params=query_params,
         )
+
+    # Tunnel methods
+
+    def create_tunnel(
+        self,
+        data: dict[str, t.Any] | CreateTunnelData,
+        *,
+        query_params: dict[str, t.Any] | None = None,
+    ) -> response.GlobusHTTPResponse:
+        """
+        :param data: Parameters for the tunnel creation
+        :param query_params: Any additional parameters will be passed through
+            as query params.
+
+        .. tab-set::
+
+            .. tab-item:: Example Usage
+
+                .. code-block:: python
+
+                    tc = globus_sdk.TunnelClient(...)
+                    result = tc.create_tunnel(data)
+                    print(result["data"]["id"])
+
+            .. tab-item:: API Info
+
+                ``POST /tunnels``
```

Comment:
I see that in all of our other, preexisting method listings, we don't show the full route. That dates from SDK v3 and earlier, when the base path was implicit. We should fix those. I think the new ones should be written in full, like so:

```suggestion
                ``POST /v2/tunnels``
```